std/utf.d

```diff
@@ -3161,7 +3161,7 @@ version(unittest)
         @property C front() { return _str[0]; }
         void popFront() { _str = _str[1 .. $]; }
 
-        this(inout(C)[] str)
+        this(const(C)[] str)
```

Comment:
What's the reason for this change?

Reply:
it leads to instantiate [`byGrapheme!(inout(char)[])`](https://auto-tester.puremagic.com/show-run.ghtml?projectid=1&runid=2986788&isPull=true) which is an impossible type ctor, but since i'll remove byGrapheme, there are string probabilities that this will not required.

Reply:
removed